### PR TITLE
Add supporters for Site-to-cloud VPN API and Click to Dial API

### PR DESCRIPTION
**Pull request by DanXu-ChinaTelecom**

### CODE REVIEW

File: APIBacklog/documentation/APIBacklog.md [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
+ | Click to Dial | China Mobile | 2023/05/12 | [Link](https://github.com/camaraproject/WorkingGroups/pull/224) | China Mobile, China Telecom, Huawei | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
  | KYC Match | KDDI | 2023/07/15 | [Link](https://github.com/camaraproject/WorkingGroups/pull/269) | KDDI, TEF, GSMA Open Gateway, Orange | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
  | KYC Fill-in | KDDI | 2023/07/15 | [Link](https://github.com/camaraproject/WorkingGroups/pull/268) | KDDI, MTN, TEF, GSMA Open Gateway, Orange | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
- | Site-to-cloud VPN | China Telecom | 2023/07/23| [Link](https://github.com/camaraproject/WorkingGroups/pull/273) | China Telecom, Huawei | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
+ | Site-to-cloud VPN | China Telecom, Huawei | 2023/07/23| [Link](https://github.com/camaraproject/WorkingGroups/pull/273) | China Telecom, Huawei, China Mobile | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
```
Comment (jordonezlucena) [inline APIBacklog/documentation/APIBacklog.md:29]: If China Mobile wants to become supporter, then a delegate from China Mobile should submit the PR.

**Review by jordonezlucena — CHANGES_REQUESTED** [review]

File: APIBacklog/documentation/APIBacklog.md (hunk 1 above)
Comment (ShutingQing) [reply APIBacklog/documentation/APIBacklog.md]: Hi Jose. Thanks. Their changes look good to me. I resolved the conflicts after Hanbai making changes. @jordonezlucena

**Review by ShutingQing — APPROVED** [review]
LGTM

**Review by hdamker — CHANGES_REQUESTED** [review]
Currently this PR would delete DT from lines 27, 28. Please don't merge like this.

**Comment by jordonezlucena** [comment]
> Currently this PR would delete DT from lines 27, 28. Please don't merge like this.

Agree. @ShutingQing : please open a new PR against the latest version of the API backlog.

**Comment by ShutingQing** [comment]
okay, i'll do that.

**Pull request closed without merging**